docs/KVStore.md

```diff
@@ -1,4 +1,6 @@
-# KVStore <!-- omit in toc -->
+# KVStore
```

Comment:
I think we should still include `<!-- omit in toc --> ` in addition to your changes so we can support all the different types of editors & plugins.